src/css/custom.css

```diff
@@ -28,3 +28,31 @@ html {
 * {
   list-style: none;
 }
+
+body {
+  font-family: 'NB International Pro';
+}
+
+@font-face {
```

Comment:
Can we move fonts to a fonts.css?

Reply:
@jjwill I initially did that during lips, but docusaurus throws an error during build when we pass in an array of css files rather than just one. If you want I can create an issue on their repo in hopes that they address it